firmware-src/pages/tryapi.html

```diff
@@ -2,23 +2,21 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="style.css">
 	<title>RESTful API</title>
-	<script type="text/javascript" src="librest.js"></script>
-	<script type="text/javascript">
-		function printinput(text) {
-			document.getElementById('input').innerHTML = text;
-		}
-
-		function print(text, err = false) {
-			document.getElementById('output').style.color = err ? 'red' : 'green';
-			document.getElementById('output').innerHTML = text;
+	<script src="libs.js"></script>
+	<style>
+	#output, #input pre { white-space: pre-wrap }
```

Comment:
I don't think so. `tryapi.html` is the only document that needs to reliably display multiline data. This style could be removed, but that would add some complexity to the rendering code.